src/app/overview/send-to-view/send-to-view.component.ts

```diff
@@ -0,0 +1,175 @@
+import { Component, OnInit } from '@angular/core';
+import { ExplorerModel } from '../../explorer/explorer.model';
+import { ExplorerService } from '../../explorer/explorer.service';
+import { WalletModel } from '../../wallet/wallet.model';
+import { WalletService } from '../../wallet/wallet.service';
+import { RpcSend } from '../../rpc/rpc-send.model';
+import { RpcReceive } from '../../rpc/rpc-receive.model';
+
+
+export interface SendToAddressModel {
+  amount: Number;
+  destinationAddress: String;
+  feeIncluded: Boolean;
+}
+
+@Component({
+  selector: 'app-send-to-view',
+  templateUrl: './send-to-view.component.html',
+  styleUrls: ['../overview.component.css','./send-to-view.component.css']
+})
+export class SendToViewComponent implements OnInit {
+
+
+  explorer: ExplorerModel;
+  wallet: WalletModel;
+  transaction: SendToAddressModel = {
+    amount: undefined,
+    destinationAddress: undefined,
+    feeIncluded: false,
+  };
+  rpcReceive: RpcReceive;
+  qrMainAddress: string;
+
+  constructor(
+    private explorerService: ExplorerService,
+    private walletService: WalletService
+  ) {}
+
+  ngOnInit() {
+    this.showUSD();
```

Comment:
This card doesn't need all this functionality, all it needs is the code for `sendToAddress`. 
The same for all the other cards, most of them only need one or two functions.